snaprecovery.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+!/bin/sh
```

Comment:
Not really sure why you removed the `#`. I'm guessing it was a mistake.

Reply:
Yep lol. I remember in vim having a # in a line and wondering where it came from.